@uportal/user-profile-menu/src/App.vue

```diff
@@ -45,6 +45,14 @@ export default {
 </script>
 
 <style>
+
+:root {
+  --user-prof-dd-item-bg-color: black;
```

Comment:
could these variables be fully written out?
For example `--user-profile-dropdown-item-background-color`